cluster-soak-tests/src/main/scala/akka/cluster/soak/ClusterSoakMain.scala

```diff
@@ -25,6 +27,9 @@ object ClusterSoakMain extends App {
   val serviceDiscovery = Discovery(system).discovery
   val resolveTimeout = 5.seconds
 
+  val dnsDispatcher = system.dispatchers.lookup("dns-dispatcher")
+  StarvationDetector.checkExecutionContext(dnsDispatcher, system.log, StarvationDetectorSettings(1.second, 5.seconds, 100.millis, 10.seconds), () => false)
```

Comment:
Might be nice to use named parameters here, not too obvious what all this does :)

Reply:
:+1: